src/button/ux-button-theme.css

```diff
@@ -76,6 +76,22 @@ styles.button.flat:active {
   background-color: ${background || $swatches.grey.p300};
 }
 
+styles.button.icon {
+  display: inline-flex;
+  align-content: center;
+  justify-content: center;
+
+  width: 40px;
+  height: 40px;
+
+  background-color: transparent;
+  color: ${foreground || $swatches.grey.p700};

```

Comment:
This usage scenario would be nice. We can circle back once we get the broader icon story resolved.